transcom/mymove
# B 21328 int

## [B-21328](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21328)

## Summary
This backlog is a quick precursor to the main feature work to ensure AC is met exactly by the larger downstream story, [B-20972](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1000658). It simply renames a Review weights button to instead read Review shipment weights

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Access the application as a TIO
2. Select any move in your queue
3. Ensure the button in the following screenshot reads "Review shipment weights"
<img width="1006" alt="image" src="https://github.com/user-attachments/assets/837d1365-138f-4bec-9289-572a93b6b0ff">


### Frontend


## Screenshots
ANDI:
<img width="1646" alt="image" src="https://github.com/user-attachments/assets/e4fbd7a6-b600-47cc-9050-5be0976e6ae9">
<img width="1646" alt="image" src="https://github.com/user-attachments/assets/667a15b4-350b-412b-9381-889bb4e69857">
<img width="1646" alt="image" src="https://github.com/user-attachments/assets/eac6110c-7f6c-406f-bbea-b49a475a3042">

Showing the unrelated element failing color contrast: 
<img width="1646" alt="image" src="https://github.com/user-attachments/assets/cf1863c4-ee44-4015-87b4-cd3910038a47">
